packages/@lwc/ssr-compiler/src/__tests__/utils/expected-failures.ts

```diff
@@ -34,12 +34,4 @@ export const expectedFailures = new Set([
     'superclass/render-in-superclass/unused-default-in-superclass/index.js',
     'wire/errors/throws-on-computed-key/index.js',
     'wire/errors/throws-when-colliding-prop-then-method/index.js',
```

Comment:
Seems there are some open questions regarding these two, which is discussed in several open issues such as:

 - https://github.com/salesforce/lwc/issues/5029
 - https://github.com/salesforce/lwc/issues/4988

Among others... So I don't think this PR will address these.